pkgs/development/python-modules/stack-data/default.nix

```diff
@@ -0,0 +1,57 @@
+{ asttokens
+, buildPythonPackage
+, executing
+, fetchgit
+, git
+, lib
+, littleutils
+, pure-eval
+, pygments
+, pytest
+, setuptools_scm
+, toml
+, typeguard
+}:
+
+buildPythonPackage rec {
+  pname = "stack_data";
+  version = "0.0.7";
+
+  # Install requires .git and fetchFromGitHub doesn't support leaving it so
+  # let's use fetchgit
+  src = fetchgit {
+    url = "https://github.com/alexmojaki/${pname}.git";
+    rev = "v${version}";
+    sha256 = "0smifhq8ma8qqhsawmgbfxq5i07rszgysf23i38dlal4h9krm03z";
+    leaveDotGit = true;
+  };
+
```

Comment:
```suggestion
  };

  SETUPTOOLS_SCM_PRETEND_VERSION = version;
```

Reply:
Ah, interesting! This `SETUPTOOLS_SCM_PRETEND_VERSION` was new to me, thanks! :+1: Fixed.